changes/ce/fix-10107.en.md

```diff
@@ -0,0 +1,9 @@
+For operations on `bridges API` if `bridge-id` is unknown we now return `404`
+instead of `400`. Also a bug was fixed that caused a crash if that was a node
+operation. Additionally we now also check if the given bridge is enabled when
+doing the cluster operation `start` . Affected endpoints:
+ * [cluster] `/bridges/:id/:operation`,
+ * [node] `/nodes/:node/bridges/:id/:operation`, where `operation` is one of
+`[start|stop|restart]`.
+Moreover for a node operation, we check if node name is in our cluster and
```

Comment:
nit: Moreover, for a node operation, we will check if the node is in our cluster, and if not, return `404` instead of `501`.